CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
   * Added `seqan3::views::char_strictly_to`. Behaves like `seqan3::views::char_to`, but throws on invalid
     input ([\#2898](https://github.com/seqan/seqan3/pull/2898)).
   * Improved performance of vector assignment for alphabets ([\#3038](https://github.com/seqan/seqan3/pull/3038)).
+  * Char literals returning std::vector are now constexpr if supported by the compiler
```

Comment:
I'll move this to the proper release when rebasing

Reply:
rebase please :)

Reply:
rebase please